packages/testcontainers/src/containers/RegTestContainer/index.ts

```diff
@@ -37,7 +37,9 @@ export class RegTestContainer extends DeFiDContainer {
       '-fortcanninghillheight=10',
       '-fortcanningroadheight=11',
       '-fortcanningcrunchheight=12',
-      '-fortcanningspringheight=13'
+      '-fortcanningspringheight=13',
+      'fortcanninggreatworldheight=14',
```

Comment:
'-fortcanninggreatworldheight=14',
'-negativeinterest=1'

Seems like these 2 flags are caushing the container to stop.

Reply:
`-negativeinterest=1` is used on ain to track negative interest, not activate it. See https://github.com/DeFiCh/ain/blob/28f2b9ea34487a087a6de82d2b0004f559a69d77/src/masternodes/govvariables/attributes.cpp#L415